plugins/in_tail/tail_db.c

```diff
@@ -95,16 +95,41 @@ int flb_tail_db_close(struct flb_sqldb *db)
     return 0;
 }
 
+static int flb_tail_db_file_delete_by_id(struct flb_tail_config *ctx,
+                                         uint64_t id)
+{
+    int ret;
+
+    /* Bind parameters */
+    sqlite3_bind_int64(ctx->stmt_delete_file, 1, id);
```

Comment:
this call should check the return value: https://www.sqlite.org/c3ref/bind_blob.html.